ludwig/schema/decoders/utils.py

```diff
@@ -36,6 +38,33 @@ def get_decoder_classes(feature: str):
     return decoder_config_registry[feature]
 
 
+@DeveloperAPI
+def get_decoder_descriptions(feature_type: str):
+    """This function returns a dictionary of decoder descriptions available at the type selection.
+
+    The process works as follows - 1) Get a dictionary of valid decoders from the decoder config registry,
+    but inverse the key/value pairs since we need to index `valid_decoders` later with an altered version
+    of the decoder config class name. 2) Loop through Decoder Metadata entries, if a metadata entry has a
+    decoder name that matches a valid decoder, add the description metadata to the output dictionary.
+
+    Args:
+        feature_type (str): The feature type to get decoder descriptions for
+    Returns:
+        dict: A dictionary of decoder descriptions
+    """
+    output = {}
+    valid_decoders = {
+        cls.module_name() if hasattr(cls, "module_name") else None: registered_name
```

Comment:
Which decoder modules don't have `module_name()`. Does it make sense to add them?